sklearn/model_selection/_split.py

```diff
@@ -632,6 +644,12 @@ def split(self, X, y, groups=None):
 
         test : ndarray
             The testing set indices for that split.
+
+        Note
+        ----
+        Multiple calls to the ``split`` method will not return identical
+        training or testing sets unless ``random_state`` is set to an integer
```

Comment:
Well, personally, I think having multiple split calls produce the same split is a better design, but not one that we currently implement.